src/common/util/include/openvino/util/file_util.hpp

```diff
@@ -12,6 +12,20 @@
 
 #include "openvino/util/util.hpp"
 
+#if defined(__linux__) && !defined(__ANDROID__)
+#    if (__cplusplus >= 201703L) || (defined(_MSVC_LANG) && (_MSVC_LANG >= 201703L) && (_MSC_VER >= 1913))
```

Comment:
Do we need check _MSVC_LANG if `#if defined(__linux__) && !defined(__ANDROID__)`?